web-workout-tracker/workout/src/assets/index.css

```diff
@@ -425,7 +429,123 @@
 }
 @media (min-width: 1918px) {
   .top-market .top-cta {
-    margin-left: 30%;
+    margin-left: 27.5%;
     margin-top: -400px;
   }
 }
+
+
+/*Team Page*/
+
+.team {
+  margin: 0 auto;
+  width: 1810px;
+  max-width: 100%;
+  display: flex; 
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding-bottom: 70px;
+}
+
+.team .member {
+  margin-top: 90px;
+  padding-top: 5px;
+  margin-left: 3%;
+  margin-right: 3%;
+  width: 30em;
+  border-radius: 10px;
+  background-color: white;
+}
+
+.member img {
+  width: 70%;
+  height: 200px;
```

Comment:
not sure, that it's a good idea to have % on width but px on height. This way on different screen resolutions image could look cool or not cool (wrong proportions). Actually on my desktop it looks not like it supposed to look

Reply:
@nia-ja I see I will fix this.

Reply:
@nia-ja Change made you can approve this PR. Everything should be fixed in my next PR.